spec/requests/core_induction_programme/course_lesson_part_spec.rb

```diff
@@ -53,6 +53,26 @@
       end
     end
 
+    describe "DELETE /core-induction-programme/years/:years_id/modules/:module_id/lessons/:lesson_id/parts/:part_id" do
+      it "deletes a course lesson part and redirects to a course lesson" do
+        post "#{course_lesson_part_url}/split", params: {
+          commit: "Save changes",
+          split_lesson_part_form: {
+            title: "Updated title one",
+            content: "Updated content",
+            new_title: "Title two",
+            new_content: "Content two",
+          },
+        }
+        delete course_lesson_part_url, params: { id: course_lesson.course_lesson_parts[1].id }
+
+        course_lesson_url = "/core-induction-programme/years/#{course_year.id}/modules/#{course_module.id}/lessons/#{course_lesson.id}"
+
+        expect(CourseLessonPart.count).to eq(0)
```

Comment:
That's the fault of destroy - but we are actually expecting one to be left :)

Cause we have two parts from the setup, and are only deleting one.